src/test/resources/application.properties

```diff
@@ -1,6 +1,7 @@
 uk.gov.companieshouse.item-handler.health = /healthcheck
 
 spring.kafka.consumer.bootstrap-servers = ${spring.embedded.kafka.brokers}
+spring.kafka.consumer.auto-offset-reset = earliest
```

Comment:
Interesting. Is there a possibility then that some test runs were failing because the consumer started consuming after the messages had been sent?